lib/ansible/modules/packaging/os/redhat_subscription.py

```diff
@@ -556,7 +556,8 @@ def _load_product_list(self, consumed=False):
             args += " --consumed"
         else:
             args += " --available"
-        rc, stdout, stderr = self.module.run_command(args, check_rc=True)
+        lang_env = dict(LANG='C', LC_ALL='C', LC_MESSAGES='C')
```

Comment:
I am not sure about preference but we have 

```
module.run_command_environ_update = dict(LANG='C', LC_ALL='C', LC_MESSAGES='C')
```

Reply:
Yeah, it's not currently needed in the rest of the module so I went with changing it only in this one place. I don't mind changing it, if you think we should do what you mentioned. ;-)